proto/prysm/v1alpha1/attestation.proto

```diff
@@ -67,31 +47,22 @@ message AggregateAttestationAndProof {
     bytes selection_proof = 2 [(ethereum.eth.ext.ssz_size) = "96"];
 }
 
-message AggregateAttestationAndProofElectra {
-    // The aggregator index that submitted this aggregated attestation and proof.
-    uint64 aggregator_index = 1  [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v5/consensus-types/primitives.ValidatorIndex"];
-
-    // The aggregated attestation that was submitted.
-    AttestationElectra aggregate = 3;
-
-    // 96 byte selection proof signed by the aggregator, which is the signature of the slot to aggregate.
-    bytes selection_proof = 2 [(ethereum.eth.ext.ssz_size) = "96"];
-}
+message Attestation {
+    // A bitfield representation of validator indices that have voted exactly
+    // the same vote and have been aggregated into this attestation.
+    bytes aggregation_bits = 1 [(ethereum.eth.ext.ssz_max) = "2048", (ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/go-bitfield.Bitlist"];
 
-message SignedAggregateAttestationAndProof {
-    // The aggregated attestation and selection proof itself.
-    AggregateAttestationAndProof message = 1;
+    AttestationData data = 2;
 
-    // 96 byte BLS aggregate signature signed by the aggregator over the message.
-    bytes signature = 2 [(ethereum.eth.ext.ssz_size) = "96"];
+    // 96 byte BLS aggregate signature.
+    bytes signature = 3 [(ethereum.eth.ext.ssz_size) = "96"];
 }
 
-message SignedAggregateAttestationAndProofElectra {
-    // The aggregated attestation and selection proof itself.
-    AggregateAttestationAndProofElectra message = 1;
-
-    // 96 byte BLS aggregate signature signed by the aggregator over the message.
-    bytes signature = 2 [(ethereum.eth.ext.ssz_size) = "96"];
+message SingleAttestation {
```

Comment:
This is an Electra type

Reply:
Fixed in 44b89cd097b9ac5e5ef016c6e79d2eaff4bafed9.